packages/teleterm/src/ui/DocumentTerminal/Terminal/xterm.ts

```diff
@@ -110,10 +132,8 @@ class TtyTerminal {
     this.term.write(displayText);
   }
 
-  _requestResize() {
-    this._fitAddon.fit();
-    this._ptyProcess.resize(this.term.cols, this.term.rows);
+  private requestResize() {
```

Comment:
Thank for making them `private`. Btw, do you think it makes sense to keep the prefixes? Seems more readable with them.

Reply:
Personally, I'm not a fan of these prefixes :) I would only use TS modifiers, these style guidelines also do not recommend underlines:
[TypeScript contributing guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines)
[AirBnB JS style guide](https://github.com/airbnb/javascript#naming--leading-underscore)
[Google TypeScript Style Guide](https://google.github.io/styleguide/tsguide.html#naming-style)